js/data/data_source.d.ts

```diff
@@ -419,3 +421,6 @@ export default class DataSource {
      */
     totalCount(): number;
 }
+
+type SearchOperations = '='|'<>'|'>'|'>='|'<'|'<='|'startswith'|'endswith'|'contains'|'notcontains';
+type EventNames = 'changed'|'loadError'|'loadingChanged'/*|'customizeLoadResult'|'customizeStoreLoadOptions'*/;
```

Comment:
@nightskylark, are these events public? `'customizeLoadResult'`, `'customizeStoreLoadOptions'`

Reply:
After discussing with Eugene and Data Grid team, I see that we don't need to publish these events, since they are designed for internal purposes only.